core/services/chainlink/testdata/config-full.toml

```
@@ -0,0 +1,275 @@
+Dev = true
+ExplorerURL = 'http://explorer.url'
+InsecureFastScrypt = true
+ReaperExpiration = '168h0m0s'
+RootDir = 'test/root/dir'
+ShutdownGracePeriod = '10s'
+FeatureFeedsManager = true
+FeatureUICSAKeys = true
+FeatureLogPoller = true
+FMDefaultTransactionQueueDepth = 100
+FMSimulateTransactions = true
+FeatureOffchainReporting2 = true
+FeatureOffchainReporting = true
+
+[Database]
+ListenerMaxReconnectDuration = '1m0s'
+ListenerMinReconnectInterval = '5m0s'
+Migrate = true
+ORMMaxIdleConns = 7
+ORMMaxOpenConns = 13
+TriggerFallbackDBPollInterval = '2m0s'
+AdvisoryLockCheckInterval = '5m0s'
+AdvisoryLockID = 345982730592843
+LockingMode = 'advisory'
```

@samsondav Wdyt about grouping these lock fields in a table?
```suggestion
[Database.Lock]
Mode = 'advisory'
AdvisoryCheckInterval = '5m0s'
AdvisoryID = 345982730592843
LeaseDuration = '5m'
LeaseRefreshInterval = '1m'
```

---

Done